.circleci/docker/Dockerfile.ubuntu1804

```diff
@@ -74,34 +76,10 @@ RUN set -ex; \
 	ar r /usr/lib/libFuzzingEngine.a *.o; \
 	rm -rf /var/lib/libfuzzer
 
-# ETHASH
-RUN set -ex; \
-	cd /usr/src; \
-	git clone --branch="v0.4.4" https://github.com/chfast/ethash.git; \
-	cd ethash; \
-	mkdir build; \
-	cd build; \
-	cmake .. -G Ninja -DBUILD_SHARED_LIBS=OFF -DETHASH_BUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX="/usr"; \
-	ninja; \
-	ninja install/strip; \
-	rm -rf /usr/src/ethash
-
-# INTX
-RUN set -ex; \
-	cd /usr/src; \
-	git clone --branch="v0.2.0" https://github.com/chfast/intx.git; \
-	cd intx; \
-	mkdir build; \
-	cd build; \
-	cmake .. -G Ninja -DBUILD_SHARED_LIBS=OFF -DINTX_TESTING=OFF -DINTX_BENCHMARKING=OFF -DCMAKE_INSTALL_PREFIX="/usr"; \
-	ninja; \
-	ninja install/strip; \
-	rm -rf /usr/src/intx;
-
 # EVMONE
-ARG EVMONE_HASH="f10d12c190f55a9d373e78b2dc0074d35d752c02cb536bb6fe754fb3719dd69e"
+ARG EVMONE_HASH="81488656a53ae1bbf186d33fc69a4f5c59d3d7419b1ba1b4832a0d409b1a33bf"
 ARG EVMONE_MAJOR="0"
-ARG EVMONE_MINOR="1"
+ARG EVMONE_MINOR="2"
 ARG EVMONE_MICRO="0"
 RUN set -ex; \
 	EVMONE_VERSION="$EVMONE_MAJOR.$EVMONE_MINOR.$EVMONE_MICRO"; \
```

Comment:
I wonder why is this docker image using wget while the others git clone?

Reply:
Probably because the release tarball doesn't contain ``libevmone-standalone.a``.

Reply:
This might be a fragment of an older version that has been carried over. Afaik, the 1804 image is not used by the fuzzer CI (and hence does not need to have libevmone-standalone.a)

Reply:
I think it's best to make it uniform i.e., a source build as in the other images

Reply:
I think it does.

Reply:
Just checked and it doesn't. @chfast any reason?

Reply:
@axic I think it's okay if the release tar ball does not contain the statically built standalone library because it is not required by builds using that docker image

Reply:
The release tar.gz package does not contain any static libraries.

Reply:
created an issue for building evmone from source for 18.04 (see #7704)